webui/react/src/pages/ExperimentDetails/ExperimentVisualization.tsx

```diff
@@ -59,6 +69,7 @@ const ExperimentVisualization: React.FC<Props> = ({
   /* eslint-disable-next-line */
   const [ batches, setBatches ] = useState<number[]>([]);
   const [ hasLoaded, setHasLoaded ] = useState(false);
+  const [ pageError, setPageError ] = useState<PageError>();
```

Comment:
praise: 👍🏼

Reply:
maybe we'd have a global hook and overlay ui for these use cases

Reply:
agreed. leave for a different PR?